src/main/java/com/robinpowered/sdk/model/Identifier.java

```diff
@@ -72,6 +72,9 @@ public static Interface fromString(String value) {
     private final String value;
     private final DateTime createdAt;
 
+    // Mutable
+    private Integer deviceId;
```

Comment:
If we simply add `deviceId` to the constructor, we're going to break everything already using the constructor without the `deviceId` parameter.

Up to you - we can change the one constructor, or create an overloaded constructor to keep BC